src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/JsonDescriptor.php

```diff
@@ -368,6 +368,19 @@ private function getCallableData($callable, array $options = array())
         if ($callable instanceof \Closure) {
             $data['type'] = 'closure';
 
+            $r = new \ReflectionFunction($callable);
+            if (false !== strpos($r->name, '{closure}')) {
```

Comment:
id say `$r->isClosure()`

but in theory this can never happen due instanceof above..

Reply:
That's not what this test is about. This *is* already reflecting a closure. Here we want to know if the closure is anonymous or not. Named closures exist when using `ReflectionMethod::getClosure()` or `Closure::fromCallable()`. This PR is *precisely* about that :)

Reply:
TIL :)